pkg/drivers/kic/oci/network_create.go

```diff
@@ -124,6 +126,15 @@ func dockerNetworkInspect(name string) (*net.IPNet, net.IP, error) {
 	return subnet, gateway, nil
 }
 
+func logDockerNetworkInspect(name string) {
+	cmd := exec.Command(Docker, "network", "inspect", name)
```

Comment:
to avoid flooding the logs and confusion, lets add a line that says, "running dubugging command to gather additional logs "

Reply:
sounds good